### PR TITLE
black the generated protocol messages code

### PR DESCRIPTION
**Pull request by altendky**

### CODE REVIEW

File: tests/util/build_network_protocol_files.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # flake8: noqa
 
 import os
+import subprocess
+import sys
```
Comment (arvidn) [inline tests/util/build_network_protocol_files.py:5]: I think you should remove the `sys` import (as highlighted by lgtm)

**Review by arvidn — DISMISSED** [review]
looks good, apart from the new lgtm issue

File: tests/util/build_network_protocol_files.py (hunk 1 above)
Comment (altendky) [reply tests/util/build_network_protocol_files.py:5]: I guess I'll have to dig into why nothing else complained about this, but yes, that is a remnant from when I tried to use `-m black`.

```suggestion
```

Comment (altendky) [reply tests/util/build_network_protocol_files.py]: Oh, the entire file got blanket `noqa`'ed.  Let's not do that either.

**Review by arvidn — APPROVED** [review]

**Comment by arvidn** [comment]
thanks for fixing this!

**Review by wjblanke — APPROVED** [review]
aok

**Pull request merged**